src/main/java/com/quadfour/dao/ITaskDAO.java

```diff
@@ -4,10 +4,10 @@
 import com.quadfour.dto.TaskDTO;
 
 public interface ITaskDAO {
-
-    boolean save(TaskDTO taskDTO) throws Exception;
+    // I know its nitpicky but this should probably be renamed to saveTask just to be clear what you are saving
```

Comment:
Don't put discussion in source code comments - that's clutter.  Discussion and rationale belongs in the GitHub pull request - I mentioned this in the instructions.